Sources/SwiftyContacts/SwiftyContacts.swift

```diff
@@ -459,13 +459,22 @@ public func isCapableToCall(completionHandler: @escaping (_ result: Bool) -> ())
     if UIApplication.shared.canOpenURL(NSURL(string: "tel://")! as URL) {
```

Comment:
this will be false when running on `macCatalyst`. is there any need for extra condition?